workflow-templates/cd.yaml

```
@@ -1,8 +1,25 @@
 # Note: additional setup is required, see https://www.jenkins.io/redirect/continuous-delivery-of-plugins
-
+#
+# Please find additional hints for individual trigger use case
+# configuration options inline this script below.
+#
 name: cd
 on:
   workflow_dispatch:
+    inputs:
+      validate_only:
+        required: false
+        type: boolean
+        description: |
+          Run validation with release drafter only.
+          => Skip the release job
```

Looks odd in the screenshot with the `.`
```suggestion
          Run validation with release drafter only
          → Skip the release job
```